hw_8_new.py

```diff
@@ -0,0 +1,56 @@
+# Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+# Декоратор має працювати для різних функцій однаково.
+import time
+def my_decorator(func):
+    def deco_func(*args, **kwargs):
+        func(*args, **kwargs)
+        print('Time of call function',func.__name__, 'is',time.strftime("%H:%M:%S"))
+    return func
```

Comment:
You didn't get the key point of decorators. Decorator takes a function and return wrapper (обгортку) for this function. This wrapper then will be called as a decorated function. To keep the original result of the decorated function it should take the same arguments and return the same value.
To pass the same arguments to the function through the wrapper we use `args and kwargs`
And to return something we need to execute wrapped function and return the value. We can do it like `return funct(*args, **kwargs)` or assign the result to some value, do execute some additional code (in your case this is print()) and return the saved value. So, for your code i should be something like:
```
res = func(*args, **kwargs)
print('Time of call function...')
return res
```

At the same time, you should return the wrapper (`deco_func`) but you shouldn't execute it since it will be executed on a function call. So, replace `return func` here to `return deco_func`